protos/file2.proto

```diff
@@ -79,30 +75,37 @@ import "google/protobuf/any.proto";
 // |     Column CN Metadata Position  |
 // |     Column CN Metadata Size      |
 // ├──────────────────────────────────┤
-// | Global Buffers                   |
-// | |C| Global Meta Buffer 0*        |
-// |     ...                          |
-// |     Global Meta Buffer GN*       |
-// ├──────────────────────────────────┤
 // | Global Buffers Offset Table      |
-// | |D| Global Buffer 0 Position*    |
+// | |C| Global Buffer 0 Position*    |
 // |     Global Buffer 0 Size         |
 // |     ...                          |
 // |     Global Buffer GN Position    |
 // |     Global Buffer GN Size        |
 // ├──────────────────────────────────┤
 // | Footer                           |
-// | A u64: Offset to C0 metadata     |
+// | A u64: Offset to column meta 0   |
 // | B u64: Offset to CMO table       |
-// | C u64: Offset to global buf 0    |
-// | D u64: Offset to GBO table       |
+// | C u64: Offset to GBO table       |
 // |   u32: Number of global bufs     |
 // |   u32: Number of columns         |
 // |   u16: Major version             |
 // |   u16: Minor version             |
 // |   "LANC"                         |
```

Comment:
Is this a breaking change in the format, given we are changing the size of the footer?

Reply:
Yes, but we haven't formally released it so I don't know that its a big issue.